guide/start/running.md

```diff
@@ -31,13 +31,13 @@ Firstly, download and install:
  * [Vagrant](http://www.vagrantup.com/downloads){:target="_blank"}
  * [Oracle VirtualBox](https://www.virtualbox.org/wiki/Downloads){:target="_blank"}
  
-Then download the default Apache Brooklyn vagrant configuration from [here](https://github.com/johnmccabe/vagrant-brooklyn-getting-started/archive/master.tar.gz){:target="_blank"}. This configuration contains everything you need to get started using Apache Brooklyn.
+Then download the provided Apache Brooklyn vagrant configuration from [here](https://www.apache.org/dyn/closer.lua?action=download&filename=brooklyn/apache-brooklyn-{{site.brooklyn-version}}/apache-brooklyn-{{site.brooklyn-version}}-vagrant.tar.gz){:target="_blank"}. This archive contains everything you need to create an environment for use with this guide, providing an Apache Brooklyn instance and some blank VMs.
```

Comment:
For me, this resolves to https://www.apache.org/dyn/closer.lua?action=download&filename=brooklyn/apache-brooklyn-0.9.0-SNAPSHOT/apache-brooklyn-0.9.0-SNAPSHOT-vagrant.tar.gz but that link gives a 404 (after redirecting to http://mirror.ox.ac.uk/sites/rsync.apache.org/brooklyn/apache-brooklyn-0.9.0-SNAPSHOT/apache-brooklyn-0.9.0-SNAPSHOT-vagrant.tar.gz). I presume that's just because it's a snapshot release?